### PR TITLE
Feat/field screens

### LINKED ISSUE #7: Dynamically Generate New Screens on the USSD response based on Survey Fields e.g. (Name, Location, Comments etc.. )
For Users to be able to provide responses to USSD Survey fields, we need to be able to display each field on a separate screen.

### PR DESCRIPTION
**Pull request by Timtech4u**
Added Screens for Survey Fields.

This closes #7

### CODE REVIEW

**Comment by Timtech4u** [comment]
This also closes #16

File: app.py [hunk 1]
```diff
@@ -33,21 +33,32 @@ def ussd_handler():
...
             response += "\n".join([ field['label'] for field in fields])
-            response += "\nEnter any key to continue or cancel."
-        
+            response += "\nEnter 0 to continue or cancel."
+            
+            if step > 1:
+                index = step - 2
+
+                if len(fields) == index :
+                    response = "END Thanks for submitting your response."
```
Comment (tuxpiper) [inline app.py:55]: Note to self: this overwrites the response that started formed back in line 47

File: app.py [hunk 2]
```diff
@@ -33,21 +33,32 @@ def ussd_handler():
...
+                    # @TODO: Send Fields Dict with Response back to API Function for Posting
+                else:
+                    response = "CON Enter Value for {} \n".format(fields[index]['label'])
```
Comment (tuxpiper) [inline app.py:58]: Note to self: this overwrites the response that started formed back in line 47

**Review by tuxpiper — APPROVED** [review]
LGTM!

**Pull request merged**